Integrations/integration-qualys.yml

```diff
@@ -1110,6 +1111,8 @@ script:
       description: Scan ID
     - contextPath: Qualys.Scan.Reference
       description: Sacn ref
```

Comment:
there's a typo here (all over the integration.
please update the description to something more informative (currently both paths has the same description)